framework/CodeInterfaces/Dymola/DymolaInterface.py

```diff
@@ -315,22 +331,23 @@ def finalizeCodeOutput(self, command, output, workingDir):
       @ In, workingDir, string, current working dir
       @ Out, output, string, optional, present in case the root of the output file gets changed in this method.
     """
-    self._vars = {}
-    self._blocks = []
-    self._namesData1 = []
-    self._namesData2 = []
-    self._timeSeriesData1 = []
-    self._timeSeriesData2 = []
+    _vars = {}
+    _blocks = []
+    _namesData1 = []
+    _namesData2 = []
+    _timeSeriesData1 = []
+    _timeSeriesData2 = []
 
     # Load the output file (.mat file) that have been generated by running Dymola executable and
     #   store the data in this file to variable 'mat'.
     matSourceFileName = os.path.join(workingDir, output)
     matSourceFileName += '.mat'
     ###################################################################
     #FIXME: LOADMAT HAS A DIFFERENT BEHAVIOR IN SCIPY VERSION >= 0.18 #
-    if int(scipy.__version__.split(".")[1])>17:
-      warnings.warn("SCIPY version >0.17.xx has a different behavior in reading .mat files!")
+    #if int(scipy.__version__.split(".")[1])>17:
```

Comment:
Is is OK to skip this check?

Reply:
yes because we realized that it is not the case with version .17